src/components/Profile/MyPosts/MyPosts.js

```diff
@@ -9,15 +9,15 @@ import AddNewPostForm from './AddNewPostForm';
 
 import style from './MyPosts.module.css';
 
-const MyPosts = (props) => {
+const MyPosts = React.memo((props) => {
```

Comment:
Consider wrapping the component in any HOC right before the export.
This way you'll have a clear component, clear component modifications and clear export.

Reply:
What do you mean?

Reply:
i do